### PR TITLE
Http status codes

### PR DESCRIPTION
**Pull request by immqu**
- harmonizes http response names across languages
- adds DFG edges from ProxiedEndpoint to HttpEndpoint
- adds a requestbody to the HttpEndpoint
- adds DFG edges from HttpEndpoint to the declaration that is initialized via a BindJSON call (in Go)

### CODE REVIEW

File: cloudpg/src/main/java/io/clouditor/graph/passes/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Developing Passes
```
Comment (oxisto) [inline cloudpg/src/main/java/io/clouditor/graph/passes/README.md:1]: I have included this already in the README. this file is not needed

File: cloudpg/src/main/java/io/clouditor/graph/passes/golang/GinGonicPass.kt [hunk 2]
```diff
@@ -44,23 +102,41 @@ class GinGonicPass : Pass() {
...
+
+                log.debug(
+                    "Adding {} to {} - resolved to {}",
+                    m.name,
+                    client?.name,
+                    endpoint.handler?.name
+                )
 
                 client?.httpEndpoints?.plusAssign(endpoint)
+                app?.functionalities?.plusAssign(endpoint)
```
Comment (oxisto) [inline cloudpg/src/main/java/io/clouditor/graph/passes/golang/GinGonicPass.kt:139]: is this needed?

Comment (immqu) [reply cloudpg/src/main/java/io/clouditor/graph/passes/golang/GinGonicPass.kt:139]: in other passes endpoints and handlers are also added as functionalities so it seemed to me that it is simply missing here

Comment (oxisto) [reply cloudpg/src/main/java/io/clouditor/graph/passes/golang/GinGonicPass.kt:139]: Ok

**Review by oxisto — APPROVED** [review]

**Pull request merged**